bowtie/_cli.py

```diff
@@ -117,27 +117,37 @@ def report(input: Iterable[str], output: TextIO):
     default=None,
     type=click.Choice(["json", "pretty"]),
 )
+@click.option(
+    "--show",
+    "-s",
+    help="Displays the summary according to Argument",
+    default="failures",
+    type=click.Choice(["failures", "validation"]),
+)
 @click.argument(
     "input",
     default="-",
     type=click.File(mode="r"),
 )
-def summary(input: Iterable[str], format: str | None):
+def summary(input: Iterable[str], format: str | None, show: str | None):
     """
     Generate an (in-terminal) summary of a Bowtie run.
     """
     if format is None:
         format = "pretty" if sys.stdout.isatty() else "json"
 
+    if show is None:
+        show = "failures" if sys.stdout.isatty() else "validation"
```

Comment:
I'm not sure deciding the default based on whether `sys.stdout` is a TTY makes sense for this one. That means deciding whether we show one or the other based on whether someone is piping `bowtie summary` into something else or not -- it makes sense for configuring output format because if stdout is not a TTY usually it means someone's storing output into a file or something, and there JSON makes more sense.

Compare e.g. what `bowtie suite -i lua-jsonschema https://github.com/json-schema-org/JSON-Schema-Test-Suite/tree/main/tests/draft4/type.json | bowtie summary` shows you to what `bowtie suite -i lua-jsonschema https://github.com/json-schema-org/JSON-Schema-Test-Suite/tree/main/tests/draft4/type.json | bowtie summary | jq` shows you, where the latter is useful if you want to query the output (say, pull out information on one implementation).

For `--show` though I think we should just pick a default that applies always -- probably `validation`.